app/components/environment_banner_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class EnvironmentBannerComponent < ViewComponent::Base
+  def render?
+    Rails.env.staging?
```

Comment:
Does `ENV["RAILS_ENV"] == "production"` on the staging environment?

https://github.com/unboxed/bops-terraform/blob/main/ansible/templates/app/etc/default/application.j2#L5

This isn't differentiated on the `ansible_env`

Reply:
We might need to set our own custom environment variable if we can't distinguish production and staging with an existing differentiated env value

Reply:
Interesting, I've always added staging environment... for staging. You have production for both?

Reply:
Yes we have production for both. If we want to have staging we will need to have an additional `staging` file [here][1]
I think we could add another environment variable maybe `ENV_NAME`. Set it to default production, on staging set it to staging and add a check if it's staging show the banner as you did! 

[1]: https://github.com/unboxed/bops/tree/main/config/environments